client/src/components/Overview/WelcomeSection.js

```diff
@@ -0,0 +1,13 @@
+const WelcomeSection = () => {
+    return (
+        <section className='c-welcome-section'>
+            <div className='o-page-content-wrapper'>
+                <h4>Good morning, Eric</h4>
```

Comment:
Shouldn't Eric be someone's first name? If so, please add a comment or some variable, thanks!

Reply:
Of course. This is just to make the design easier to imagine. When we start implementing this part, we will add the corresponding variable. I will add a comment for now. Thanks.